src/AffectedUserFlow/PublishConsent/PublishConsentForm.tsx

```diff
@@ -233,67 +239,69 @@ const PublishConsentForm: FunctionComponent<PublishConsentFormProps> = ({
   )
 }
 
-const style = StyleSheet.create({
-  outerContainer: {
-    flex: 1,
-    backgroundColor: Colors.primaryLightBackground,
-  },
-  navButtonContainer: {
-    flexDirection: "row",
-    justifyContent: "space-between",
-    position: "absolute",
-    paddingTop: 25,
-    width: "100%",
-    borderBottomWidth: Outlines.hairline,
-    borderBottomColor: Colors.neutral10,
-    backgroundColor: Colors.primaryLightBackground,
-    zIndex: Layout.zLevel1,
-  },
-  backButtonInnerContainer: {
-    padding: Spacing.medium,
-  },
-  cancelButtonInnerContainer: {
-    padding: Spacing.medium,
-  },
-  contentContainer: {
-    paddingHorizontal: Spacing.large,
-    paddingTop: 105,
-    paddingBottom: Spacing.huge,
-  },
-  content: {
-    marginBottom: Spacing.small,
-  },
-  header: {
-    ...Typography.header1,
-    paddingBottom: Spacing.medium,
-  },
-  subheaderText: {
-    ...Typography.body1,
-    ...Typography.mediumBold,
-    color: Colors.black,
-    marginBottom: Spacing.xxSmall,
-  },
-  bodyText: {
-    ...Typography.body1,
-    marginBottom: Spacing.xxLarge,
-  },
-  button: {
-    alignSelf: "flex-start",
-  },
-  bottomButtonContainer: {
-    backgroundColor: Colors.secondary10,
-    flexDirection: "row",
-    alignItems: "center",
-    justifyContent: "center",
-    paddingVertical: Spacing.small,
-    borderTopColor: Colors.neutral25,
-    borderTopWidth: Outlines.hairline,
-  },
-  bottomButtonText: {
-    ...Typography.header5,
-    color: Colors.primary100,
-    marginRight: Spacing.xSmall,
-  },
-})
+const createStyle = (insets: EdgeInsets) => {
```

Comment:
would it make sense to have this function be more specific to the particular component that is dependent on the parameter? something like
```typescript
const createButtonContainerStyle = (insets: EdgeInsets) => {
....

const style = StyleSheet.create({
...
```

Reply:
What do you think the advantage of that would be? I'm leaning toward keeping all the styles in one place for simplicity, and because naming could get weird if we have more than one component that uses the parameter.